guides/meeting_events.md

```diff
@@ -175,4 +175,77 @@ The following table lists available states.
 |`videoInputSelected`|The camera was selected.
 
 ## Example
-[The Chime SDK serverless demo](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless) uses Amazon CloudWatch Logs to collect, process, and analyze meeting events. For more information, see [the Meeting Dashboard section](https://github.com/aws/amazon-chime-sdk-js/tree/master/demos/serverless#meeting-dashboard) on the serverless demo page.
\ No newline at end of file
+
+This section includes sample code for the [Monitoring and troubleshooting with Amazon Chime SDK meeting events](https://aws.amazon.com/blogs/business-productivity/monitoring-and-troubleshooting-with-amazon-chime-sdk-meeting-events/) blog post.
+
+1. Follow the [blog post](https://aws.amazon.com/blogs/business-productivity/monitoring-and-troubleshooting-with-amazon-chime-sdk-meeting-events/) to deploy the AWS CloudFormation stack. The stack provisions all the infrastructure required to search and analyze meeting events in Amazon CloudWatch.
+2. To receive meeting events in your iOS application, add an event analytics observer to implement the `onEventReceived` method.
```

Comment:
`onEventReceived` is not consistent with `eventDidReceive` in the code snippet.

Reply:
A nice catch! I updated it to the `eventDidReceive` method.